wsd/annotate/app.py

```diff
@@ -48,7 +44,7 @@ def get_next_document():
     try:
         linhub.document = linhub.get_next_document(LINHUB_TASK)
         return linhub.document
-    except:
+    except ValueError:
```

Comment:
I don't think ValueError is the right exception here. Request should return a 404 when no more documents are available, indicating the task is completed.